discovery-provider/src/queries/get_health.py

```diff
@@ -667,3 +679,23 @@ def get_latest_chain_block_set_if_nx(redis=None, web3=None):
             )
 
     return latest_block_num, latest_block_hash
+
+
+def get_backfill_health_info():
+    db = db_session.get_db_read_replica()
+    solana_client_manager = SolanaClientManager(shared_config["solana"]["endpoint"])
```

Comment:
This breaks with this error, can anyone tell me why?
`{"levelno": 40, "level": "ERROR", "msg": "Non Audius-derived exception\nTraceback (most recent call last):\n  File \"/usr/lib/python3.9/site-packages/flask/app.py\", line 1838, in full_dispatch_request\n    rv = self.dispatch_request()\n  File \"/usr/lib/python3.9/site-packages/flask/app.py\", line 1824, in dispatch_request\n    return self.view_functions[rule.endpoint](**req.view_args)\n  File \"/audius-discovery-provider/src/queries/health_check.py\", line 58, in health_check\n    (health_results, error) = get_health(args)\n  File \"/audius-discovery-provider/src/queries/get_health.py\", line 207, in get_health\n    backfill_health_info = get_backfill_health_info()\n  File \"/audius-discovery-provider/src/queries/get_health.py\", line 688, in get_backfill_health_info\n    user_bank = index_user_bank_backfill_health_check(\n  File \"/audius-discovery-provider/src/tasks/index_user_bank_backfill.py\", line 514, in check_if_backfilling_complete\n    one_sig_before_stop = solana_client_manager.get_signatures_for_address(\n  File \"/audius-discovery-provider/src/solana/solana_client_manager.py\", line 111, in get_signatures_for_address\n    return _try_all_with_timeout(\n  File \"/audius-discovery-provider/src/solana/solana_client_manager.py\", line 204, in _try_all_with_timeout\n    raise Exception(message)\nException: solana_client_manager.py | get_signatures_for_address | All requests failed", "timestamp": "2022-11-03 09:02:18,185", "pathname": "/audius-discovery-provider/src/app.py", "funcName": "handle_exception", "lineno": 303, "service": "server", "otelSpanID": "e3d7ae7fe1391fcc", "otelTraceID": "08a3b436ad5a28ea017f206b5e66503e", "otelServiceName": "discovery-provider"}`